src/main/java/com/minecolonies/coremod/inventory/AbstractInteractiveItemStackHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.minecolonies.coremod.inventory;
+
+import net.minecraftforge.items.ItemStackHandler;
+
+/**
+ * Abstract itemstack handler that implements IInteractiveItemHandler.
+ */
+public abstract class AbstractInteractiveItemStackHandler extends ItemStackHandler implements IInteractiveItemHandler {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)